### PR TITLE
Peer Code Review Nov 2nd

### PR DESCRIPTION
**Pull request by chaarz**
Let's work together and get a better grade!

### CODE REVIEW

File: src/components/User.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import { useParams } from "react-router-dom";
+
+const User = () => {
+  //Show all the thoughts that have been posted for the uid
+  const { userId } = useParams();
+  const [userThought, setUserThought] = useState([]);
+  const database = getDatabase(firebaseConfig);
+  const dbRef = ref(database);
+
+  useEffect(() => {
```
Comment (chaarz) [inline src/components/User.js:13]: there is the same chunk of code in the StoredThoughts.js component - maybe try to rewrite it using a variable that you can use in both components instead?

File: src/components/apis/ImgSearch.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  .then((res) => {
+    setImage(res.data.urls.regular);
+    setAlt(res.data.alt_description);
+  })
+  //handle error from Axios docs
+  .catch (function (error) {
+    if (error.response) {
+      // The request was made and the server responded with a status code
+      // that falls out of the range of 2xx
+      console.error(error.response.data);
```
Comment (chaarz) [inline src/components/apis/ImgSearch.js]: replace console.error with an alert()

Comment (luu-jennifer) [reply src/components/apis/ImgSearch.js]: Ok, thanks! Working on updating it.

File: src/partials/_mediaQueries.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      label {
+        font-size: 1.25rem;
+      }
+      p {
+        font-size: 1rem;
+        width: 100%;
+      }
+    }
+  }
+  .storedThoughts ul li {
```
Comment (chaarz) [inline src/partials/_mediaQueries.scss:30]: the user name is not showing up properly (overflow and breaks into 2 lines ) below 1400px

**Review by chaarz — COMMENTED** [review]
Hey Jen - can you look at the comments I left on your code! Thanks

File: src/components/apis/ImgSearch.js (hunk 2 above)
Comment (luu-jennifer) [reply src/components/apis/ImgSearch.js]: Thanks, just updated.

File: src/components/User.js (hunk 1 above)
Comment (luu-jennifer) [reply src/components/User.js:13]: Thanks Char, I'll make this as a stretch goal.

File: src/partials/_mediaQueries.scss (hunk 3 above)
Comment (luu-jennifer) [reply src/partials/_mediaQueries.scss:30]: Fixed.

**Pull request merged**